tox.ini

```diff
@@ -6,7 +6,7 @@ skipsdist = True
 [testenv]
 install_command = pip install {opts} {packages}
 deps =
-    -c{env:TOX_CONSTRAINTS_FILE:https://releases.openstack.org/constraints/upper/yoga}
```

Comment:
This should work if you've cleared your pip cache

Reply:
Reverted